config/peerpods/podvm/osc-podvm-create-job.yaml

```diff
@@ -93,3 +95,5 @@ spec:
         - name: store-volume
           emptyDir: {}
       restartPolicy: Never
+      imagePullSecrets:
+        - name: 12445777-pull-request-test-pull-secret
```

Comment:
still here?

Reply:
probably cache. The file is outdated here, but if you go to the file is gone.